recipes-ros/mavros/mavros.inc

```diff
@@ -0,0 +1,15 @@
+#
+# MAVROS -- MAVLink extendable communication node for ROS with proxy for Ground Control Station.
+#
+
+SRCBRANCH = "indigo-devel"
+SRC_URI = "git://github.com/mavlink/mavros.git;protocol=http;branch=${SRCBRANCH}"
+SRCREV = "64031594d569e23ab0ddb50bf9ff195cc4c70336"
+
+DEPENDS += "std-msgs message-runtime message-generation geometry-msgs rosconsole-bridge mavlink"
```

Comment:
I am not a fan of having common dependencies in the .inc file. The scattering of the dependencies creates more confusion than the effort of maintaining the duplicated information.